src/test/java/Homework17jimmyvu.java

```diff
@@ -0,0 +1,42 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework17jimmyvu extends BaseTest{
+    @Test(enabled = true, priority = 1, description = "add a song to a playlist")
+    public static void addSongToPlaylist() throws InterruptedException {
+        navigateToPage();
+        Thread.sleep(1000);
+        login("jimmypvu@gmail.com", "te$t$tudent");
+        Thread.sleep(1000);
+        getASong();
+        Thread.sleep(1000);
+
+        WebElement song = driver.findElement(By.xpath("//section[@id='playlistWrapper']/div[@class='song-list-wrap main-scroll-wrap playlist']/div[@class='virtual-scroller scroller']/div[@class='item-container']/table[@class='items']/tr[@class='song-item']/td[contains(text(), 'Waiting On A Train')]"));
```

Comment:
**The following principle must be held in deciding locators: the locator must be**

- unique
- readable 
- maintainable

Try this locator `//tr[@class='song-item selected' ]//td[contains(text(), 'Waiting On A Train')]`

![image](https://user-images.githubusercontent.com/104761482/210968173-b7248146-08e1-4354-bf69-a2600795f55e.png)